src/libraries/System.Formats.Tar/src/System/Formats/Tar/PosixTarEntry.cs

```diff
@@ -62,7 +62,10 @@ public int DeviceMajor
                 }
```

Comment:
Can't comment above this line: Can you please also update the exception message in line 53?